src/rp2_common/hardware_clocks/include/hardware/clocks.h

```diff
@@ -89,6 +89,46 @@ extern "C" {
 #define KHZ 1000
 #define MHZ 1000000
 
+/// \tag::pll_settings[]
```

Comment:
note the "tag" here hoists the comment into the documentation, so it may be worth checking whether this new content make sense in that context (cc @lurch)

Reply:
The documentation just grabs whatever appears between the start and end tags - see page 232 of https://datasheets.raspberrypi.com/rp2040/rp2040-datasheet.pdf for where this tag-content is displayed currently (and its surrounding context).
So yeah the "settings below" part should probably be moved outside of the tags.